src/mongocxx/error/write_concern.hpp

```diff
@@ -1,38 +0,0 @@
-// Copyright 2014 MongoDB Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License./**
-
-#include <mongocxx/config/prelude.hpp>
-
-#include <cstdint>
-#include <string>
-
-// TODO: figure out if we need this
```

Comment:
It looks like this is an earlier version of error handling. This file was deleted as part of Matt's pull request.